presto-main/src/main/java/com/facebook/presto/operator/aggregation/ApproximateLongPercentileAggregations.java

```diff
@@ -130,4 +134,9 @@ public static void output(DigestAndPercentileState state, BlockBuilder out)
             BIGINT.writeLong(out, digest.getQuantile(percentile));
         }
     }
+
+    private static void checkIfWeightGreaterThanZero(long weight)
```

Comment:
@maciejgrzybek it is suggested in the issue that error message shouldn't be repeated.